Javascript/js-variables-and-numbers_social-media-post/js/index.js

```diff
@@ -9,14 +9,26 @@ console.clear();
 	- a boolean called isReported
 */
 
-// --v-- write your code here --v--
+const title = "Media Post";
+const text = "Explore my media post";
+let numberOfLikes = 50;
```

Comment:
The code works fine, but if you read the instructions carefully, you was supposed to declare this as a const variable here and then adapt the code by task number2.